core/src/tangram.h

```diff
@@ -11,12 +11,27 @@ namespace Tangram {
 
 class DataSource;
 
+enum LabelType {
+    icon,
+    text,
+};
+
 struct TouchItem {
     std::shared_ptr<Properties> properties;
     float position[2];
     float distance;
```

Comment:
I left it for now because of the potential thumb size that we would introduce. Right now we only return one item since the feature querying is pixel perfect, but we would potentially have a distance from the center of thumb.

I will clean that up and we can rethink this more when we introduce that option.